### DISCUSSION TITLE
11번째 미팅 때 논의하고 싶은 내용

### DISCUSSION THREAD

**Discussion by Joseph-Cha** [Joseph-Cha/JustFight_Client#59] [Q&A]
- PR 리뷰 
    - #51 
    - #54 
- 프로젝트 스케줄 조정
    - 클라이언트 구현 -> 플레이어 이동까지 구현 (~8월 말)
    - 서버 구현 -> (~12월 말)
- 서버 구현
    - 현재 서버 강의 수강하는 속도를 고려했을 때 8월말 9월초에 완강 예정
    - 서버 강의를 들으면서 만든 서버를 바탕으로 이곳 프로젝트에 넣는 방식으로 구현할 예정

**Comment by jongfeel** [comment]
### 서버 요구사항 정리

- 기능 중심으로 간단하게
- 어떤 데이터들이 동기화 될 것인지 정하기

### 멘토링 주기 변경

- 2주에 한번 평일

### 멘토링 방식에 대한 생각 변화

- 어느 정도 해 놓고 리뷰 받자 (X)
- 내 페이스대로 진행하고 어느 순간 중간에 리뷰가 진행됨 (O)

요렇게 생각해 보면 좋을 것 같습니다.

아니면 지금 하는 방식이기는 한데 2주에 한번 어느 날을 정해서 리뷰 하자고 하면 하는 방식도 좋을 것 같아요.

### 다음 프로젝트를 한다면?

책을 읽고 논의를 하는 북토크 형태로 진행이 되면 좋겠음
